pytools/__init__.py

```diff
@@ -1489,10 +1489,12 @@ class Table:
     .. automethod:: add_row
     .. automethod:: __str__
     .. automethod:: latex
+    .. automethod:: github_markdown
     """
 
-    def __init__(self):
+    def __init__(self, alignments=['l', 'r']):
```

Comment:
I don't like that default: What if the table has more than two columns?

Reply:
Added in the doc - when a column has no alignment specified, it will use the last alignment in the list. In the default case shown here, all columns after the second will also have an 'r' alignment.